### PR TITLE
Badges!

### LINKED ISSUE #208: Badges!
- [pypi](https://pypi.org/project/rubrix/)
- [Code style: black](https://github.com/ambv/black)
- [Recognai Twitter](https://twitter.com/recogn_ai)
- Docker Hub badges: number of pulls and release
- Codecov

### PR DESCRIPTION
**Pull request by ignacioct**
Closes #208 

Feel free to suggest a different disposition of badges! Also, I was very surprised by how many Docker pulls we have, but I don't exactly know how they are measured.

### CODE REVIEW

**Comment by ignacioct** [comment]
Well, after fighting a little bit against Github Actions, I'm obtaining the following error on the calculation of the coverage report.

```
=========================== short test summary info ============================
379
ERROR temporal/test_app.py
380
ERROR tests/test_init.py
381
ERROR tests/test_log.py
382
ERROR tests/test_logging.py
383
ERROR tests/client/test_asgi.py
384
ERROR tests/client/test_client.py
385
ERROR tests/client/test_models.py
386
ERROR tests/server/datasets/test_api.py
387
ERROR tests/server/datasets/test_dao.py
388
ERROR tests/server/datasets/test_model.py
389
ERROR tests/server/info/test_api.py
390
ERROR tests/server/snapshots/test_api.py
391
ERROR tests/server/text_classification/test_api.py
392
ERROR tests/server/text_classification/test_model.py
393
ERROR tests/server/token_classification/test_api.py
394
ERROR tests/server/token_classification/test_model.py
395
!!!!!!!!!!!!!!!!!!! Interrupted: 16 errors during collection !!!!!!!!!!!!!!!!!!!
396
============================== 16 errors in 1.66s ==============================
397
Error: Process completed with exit code 2.
```

@frascuchon , could these tests be the ones that need a pre-existing environment to work? How can we make so CodeCov accepts them?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -54,6 +54,28 @@ jobs:
...
+      - name: Checkout Code 🛎
+        uses: actions/checkout@v2
+      - name: Setup Python 🐍
+        uses: actions/setup-python@master
+        with:
+          python-version: 3.7
+      - name: Generate Report 📄
+        run: |
+          pip install pytest-cov
+          pytest --cov=./
```
Comment (dcfidalgo) [inline .github/workflows/ci.yml]: I think we should avoid running our test suite twice, see the job above. So maybe include the coverage generation/uploading in the job above.

**Review by dcfidalgo — CHANGES_REQUESTED** [review]
Nice!! Just a small comment about efficiency, I think we should avoid running our test suite twice.

**Comment by ignacioct** [comment]
I've submitted a ticket to the Support Forum os Codecov, as I'm totally lost on why our test coverage report is not being correctly uploaded.

**Comment by frascuchon** [comment]
Nice work !!! 

I've testing badges info and I found code coverage is not reported:

Screenshot 2021-07-15 at 17 00 40


Maybe the generated coverage xml report and the  configured files should be the same? 

Screenshot 2021-07-15 at 16 59 29


Screenshot 2021-07-15 at 16 59 03

**Review by dcfidalgo — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,18 +1,36 @@
...
+    <a href="https://hub.docker.com/r/recognai/rubrix">
+        <img alt="CI" src="https://img.shields.io/docker/pulls/recognai/rubrix">
+    </a>
+    <a href="https://hub.docker.com/r/recognai/rubrix">
+        <img alt="CI" src="https://img.shields.io/docker/v/recognai/rubrix?sort=semver">
+    </a>
+    <img alt="Codecov" src="https://img.shields.io/codecov/c/github/recognai/rubrix">
 </p>
 
+<h3 align="center">Explore, label, and monitor data for AI projects</h3>
```
Comment (dvsrepo) [inline README.md]: I'd place this h3 before the badges, if possible

**Review by dvsrepo — CHANGES_REQUESTED** [review]

**Comment by dcfidalgo** [comment]
Ok, i will merge this PR, which should upload a first cov report. The coverage is quite bad, mainly because of the `sdk` folder. Since the sdk code is created automatically (i think almost entirely), i am not sure if we need tests for it. We could exclude it in the cov report, @frascuchon what do you think?

**Comment by frascuchon** [comment]
We could filter sdk folder for coverage reports. Can we configure folder/module exclusion for codecov @dcfidalgo?

**Pull request merged**